packages/account/src/Components/self-exclusion/self-exclusion-article-content.tsx

```diff
@@ -41,58 +31,41 @@ export const selfExclusionArticleItems = ({
                                 href={getStaticUrl('/responsible', { is_deriv_crypto })}
                             />,
                         ]}
-                        values={{ brand_website_name: getBrandWebsiteName() }}
                     />
                 ),
             },
-        ];
-
-        if (is_uk) {
```

Comment:
Removing the `is_uk` check in self_exclusion content since we are not supporting UK business anymore